cloud/scope/vcn_reconciler.go

```diff
@@ -62,19 +62,13 @@ func (s *ClusterScope) GetVcnName() string {
 	return fmt.Sprintf("%s", s.OCIClusterAccessor.GetName())
 }
 
-func (s *ClusterScope) GetVcnCidr() string {
-	if s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDR != "" {
-		return s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDR
+func (s *ClusterScope) GetVcnCidrs() []string {
+	if s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDRS != nil && len(s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDRS) > 0 {
+		return s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDRS
+	} else if s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDR != "" {
+		return []string{s.OCIClusterAccessor.GetNetworkSpec().Vcn.CIDR}
```

Comment:
NIT: since CIDRS supercedes CIDR thought on calling this out in variable description in types.go?

Reply:
we have called this out in types.go? like 	// Deprecated, please use NetworkDetails.cidrs
?

Reply:
regarding DNS bug, please see the diff where we were creating a new spec, but ignoring setting the dns label